crates/hdk/src/link.rs

```diff
@@ -1,5 +1,17 @@
 use crate::prelude::*;
 
+#[repr(u8)]
+pub enum HdkLinkType {
+    Path = u8::MAX - 1,
+    Default = u8::MAX,
```

Comment:
I really don't love that these two variants are words reserved by the std lib. Could we choose different words like:
```rust
Paths = u8::MAX - 1,
Any = u8::MAX,
```